apps/backend/src/auth/auth.service.ts

```diff
@@ -1,12 +1,30 @@
 import { Injectable } from '@nestjs/common';
 import { JwtService } from '@nestjs/jwt';
+import { UsersService } from '../app/users/users.service';
 import { User } from './types/user';
 
 @Injectable()
 export class AuthService {
-  constructor(private readonly jwtService: JwtService) {}
+  constructor(private readonly usersService: UsersService, private readonly jwtService: JwtService) {}
 
   generateJwt(user: User) {
     return this.jwtService.sign({ ...user, typ: 'Bearer' });
   }
+
+  async validateUser(email: string) {
+    const user = await this.usersService.findByEmail(email);
```

Comment:
I am not sure why `user` don't have `password` property here

Reply:
This was playground to improve in the feature.

Reply:
@KonradKarlinski  So, I need to add a `password` property, right ?